src/pages/bridge/Issue/index.tsx

```diff
@@ -58,6 +56,16 @@ function Issue(props: IssueProps): JSX.Element {
     return amount ? decimalToStellarNative(amount) : Big(0);
   }, [amount]);
 
+  const disclaimerText = useMemo(
+    () =>
+      `• Issue Fee: ${issueFee.mul(100)}% of the transaction amount.
+       • Redeem Fee: ${redeemFee.mul(100)}% of the transaction amount.
+       • Security deposit: ${issueGriefingCollateral.mul(100)}% of the transaction amount.
+       • Total issuable amount (in USD): 20,000 USD.
```

Comment:
Nit: If we say it's 'in USD' in the brackets, we don't need to mention it again after the value. 
```suggestion
       • Total issuable amount (in USD): 20,000.
```

Reply:
I prefer redundancy here.